assets/css/style.css

```diff
@@ -7,5 +7,5 @@ body {
 
 div.ui.divbg.container {
     background-color:white;
-    padding:1em;
+    padding:0.25em;
```

Comment:
Why was this changed?

Reply:
It was changed cause 1em was too wide and messed up the footer by adding a white background around it which was too big and made it look out of place.

Reply:
try changing this to `padding-top:1em;`